test/ttmlir/Dialect/TTNN/embedding/embedding_1d_tensor.mlir

```diff
@@ -1,4 +1,5 @@
 // RUN: ttmlir-opt --ttir-load-system-desc --ttir-layout --ttnn-open-device --convert-ttir-to-ttnn %s | FileCheck %s
+// UNSUPPORTED: true
```

Comment:
I'm a little confused as to what this change is trying to achieve. We're adding support for 1d tensor but have to disable a previously working 1d tensor test?